Drop-In/src/test/java/com/braintreepayments/api/DropInClientUnitTest.java

```diff
@@ -56,6 +58,18 @@ public void beforeEach() {
         activity = activityController.get();
     }
 
+    @Test
+    public void constructor_setsIntegrationTypeDropIn() {
+        DropInClient sut = new DropInClient(ApplicationProvider.getApplicationContext(), Fixtures.TOKENIZATION_KEY, new DropInRequest());
+        assertEquals(IntegrationType.DROP_IN, sut.braintreeClient.getIntegrationType());
+    }
+
+    @Test
+    public void constructor_setsBraintreeClientWithSessionId() {
+        DropInClient sut = new DropInClient(ApplicationProvider.getApplicationContext(), Fixtures.TOKENIZATION_KEY, new DropInRequest());
+        assertNotNull(sut.braintreeClient.getSessionId());
```

Comment:
Can we do better than a `notNull` assert? I assume no, since we'd have to mock the analytics layer which is in `core`. Just wanted to poke on it though

Reply:
The public constructor generates a random ID. We added an additional test for the internal constructor that can take a session ID as an argument to verify that it's the same.